src/services/completions.ts

```diff
@@ -358,6 +377,28 @@ namespace ts.Completions {
         return undefined;
     }
 
+    function addLabelStatementCompletions(node: Node, entries: CompletionEntry[], uniques = createMap<true>()): void {

```

Comment:
This really only depends on the node. You can remove the `uniques ` parameter and just use `const uniques = createMap<true>`, and remove the `entries` parameter and just return an array.